docs/zh/deploy/install_deploy.md

```diff
@@ -4,7 +4,17 @@
 
 ### 操作系统
 
-发行的预编译包支持：CentOS 7.x, Ubuntu 20.04, SUSE 12 SP3, macOS 12。对于其他操作系统发行版本，预编译包未做充分测试，无法保证其完全兼容。你可以尝试 [从源码编译](compile.md)，来支持其他的操作系统。
+发行的预编译包支持：CentOS 7.x, Ubuntu 20.04, SUSE 12 SP3, macOS 12。因此Linux要求glibc version >= 2.17。对于其他操作系统发行版本，预编译包未做充分测试，无法保证其完全兼容。你可以尝试 [从源码编译](compile.md)，来支持其他的操作系统。
+
+```{note}
+Linux可通过以下命令来检查系统的支持情况。
+````shell
+cat /etc/redhat-release # redhat only
```

Comment:
`/etc/os-release` exists in most Linux Distro

Reply:
Yes, but some don't have it. I'll add it, user can run all, at least one file works